docs/user-guide/disassembler-backends/binary_ninja.md

```diff
@@ -2,44 +2,60 @@
 
 ## Install
 
-Binary Ninja is not distributed with OFRAK. **You need to have a valid headless BinaryNinja license to build and run the image.** For more details:
-- [Read about the environment setup](../../environment-setup.md#binary-ninja).
-- See the [Docker commands that are run](https://github.com/redballoonsecurity/ofrak/blob/master/disassemblers/ofrak_binary_ninja/Dockerstub).
+Binary Ninja is not distributed with OFRAK. **You need to have a valid Binary Ninja license to use OFRAK Binary Ninja.** You can run OFRAK Binary Ninja natively with a valid **commercial** licence, and in a Docker container with a valid **headless** license.
 
-The recommended BinaryNinja version to use with OFRAK is 3.2.3814. If you are running OFRAK outside of the Docker image, you can switch to this version of BinaryNinja using the [BinaryNinja version switcher](https://github.com/Vector35/binaryninja-api/blob/dev/python/examples/version_switcher.py).
+The recommended Binary Ninja version to use with OFRAK is 3.2.3814. If you are running OFRAK outside of the Docker image, you can switch to this version of Binary Ninja using the [Binary Ninja version switcher](https://github.com/Vector35/binaryninja-api/blob/dev/python/examples/version_switcher.py).
 
+=== "Native"
```

Comment:
Not sure what's with the equals signs here (and below), but this does not seem to be valid Markdown?

Reply:
Are you running this with the latest code from master? There was a mkdocs extension added in the Ghidra PR that has now been merged into main called `pymdownx.tabbed` which creates tabs when there are triple equal signs. The tabbed content is necessary for it to render properly.

Reply:
You are right! I had an incorrect version of MkDocs running locally. Resolving this and the subsequent comment. My bad!